config/routes.rb

```diff
@@ -1,11 +1,13 @@
 Rails.application.routes.draw do
+  get 'login/index'
   get 'about/Index'
   root 'home#index'
-  resources :users
+  resources :users, :login
   get '/index', to:'home#index'
   get '/signup', to:'users#user'
   post '/create', to:'users#create'
   get '/articles', to:'main#index'
-  post '/login', to:'sessions#create'
-  get '/about', to:'about#Index'
+  post '/log', to:'login#create'
```

Comment:
Perhaps you could change this to also be login as it would be clearer.

Reply:
Same as above, changed the name